test/libsolidity/SolidityParser.cpp

```diff
@@ -1164,6 +1164,56 @@ BOOST_AUTO_TEST_CASE(constant_is_keyword)
 	CHECK_PARSE_ERROR(text, "Expected identifier");
 }
 
+BOOST_AUTO_TEST_CASE(keyword_is_reserved)
```

Comment:
Can this test case be extracted and merged with current keywords?

Reply:
OK, see #3633